### PR TITLE
Added losetup command

### PR DESCRIPTION
**Pull request by kmyounes**
For issue #1316

### CODE REVIEW

File: pages/linux/losetup.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+> Set up and control loop devices.
+
+- List loop devices with detailed info:
+
+`losetup -a`
+
+- Associate / Attach a device or file to a {{loop}} device:
+
+`sudo losetup /dev/{{loop}} /{{path/to/<file>}}`
```
Comment (waldyrious) [inline pages/linux/losetup.md]: This deviates slightly from how we use tokens. I'd suggest something along these lines:

    - Attach a file to a given loop device:
    
    `sudo losetup /dev/{{loop_device}} /{{path/to/file}}`

File: pages/linux/losetup.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+- Detach All {{loop}} devices:
```
Comment (waldyrious) [inline pages/linux/losetup.md]: This can be simply 

    - Detach all loop devices:

File: pages/linux/losetup.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+
+`sudo losetup -D`
+
+- Detach a specific {{loop}} device:
```
Comment (waldyrious) [inline pages/linux/losetup.md]: We don't normally use the token syntax in the example descriptions. The word "specific" (or sometimes "given") provides sufficient context for connecting the tokens in the example with the corresponding parts of the description.

File: pages/linux/losetup.md [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+
+- List loop devices with detailed info:
+
+`losetup -a`
+
+- Attach a file to a given loop device:
+
+`sudo losetup /dev/{{loop}} /{{path/to/<file>}}`
+
+- Detach All loop devices:
```
Comment (sbrl) [inline pages/linux/losetup.md]: `All` shouldn't have a capital letter at the beginning.

Comment (kmyounes) [inline pages/linux/losetup.md]: Thank you, you can mount iso,img and many others. 
Also, you can make images with dd and use them with losetup.

Comment (sbrl) [inline pages/linux/losetup.md]: Oooh awesome! I'll have to play around with it.

**Comment by waldyrious** [comment]
I've also made a minor change (we don't use `<>` to indicate tokens, as described [here](https://github.com/tldr-pages/tldr/issues/731#issuecomment-181001807)).

Let's see if squash-merging using github's web interface preserves authorship.

**Comment by waldyrious** [comment]
> Let's see if squash-merging using github's web interface preserves authorship.

Cool, it works :)

**Comment by kmyounes** [comment]
First contribution 🥇 , feel happy :)

**Comment by sbrl** [comment]
@waldyrious Yep, it does! I've been using the web interface for everything here I've done so far :wink:

**Pull request merged**